### ISSUE TITLE
passing input data 

### ISSUE THREAD

**Issue by riyajatar37003** [triton-inference-server/server#7229]
i have two models inside triton model repository. one is bert based model other is python backend which handles all preprocessing,postprocessing and all . as of now don't have ensemble implementation.

**config for python backend**
`backend: "python"
max_batch_size: 1

input [
  {
    name: "request"
    data_type: TYPE_STRING
    dims: [ 1 ]

  },
  {
    name: "want_request"
    data_type: TYPE_STRING
    dims: [ 1 ]
    optional: true
  },
  {
    name: "options"
    data_type: TYPE_STRING
    dims: [ 1 ]
    optional: true
  }
]

output [
  {
    name: "response"
    data_type: TYPE_STRING
    dims: [-1]
  },
  {
    name: "error"
    data_type: TYPE_STRING
    dims: [-1]
  },
  {
    name: "response_metadata"
    data_type: TYPE_STRING
    dims: [-1]
  }
]

instance_group [
  {
    count: 12
    kind: KIND_CPU
  }
]

dynamic_batching {
  default_queue_policy {
    max_queue_size: 1
  }
}

response_cache {
  enable: false
}`

**config for bert based model**
`name: "bert_based_model"
backend: "onnxruntime"
max_batch_size: 128
dynamic_batching {max_queue_delay_microseconds: 10000}
input [
  {
    name: "input_ids"
    data_type: TYPE_INT64
    dims: [ 1 ]
  },
  {
    name: "attention_mask"
    data_type: TYPE_INT64
    dims: [ 1 ]
  }
]
output [
  {
    name: "logits"
    data_type: TYPE_FP32
    dims: [1]
  }
]
instance_group [
{
count: 1
kind: KIND_GPU
}
]`

**Describe the solution you'd like**
I am using model-analyzer to benchmark different configuration . but i am getting input data error after running below command
`model-analyzer -v profile -f config.yaml --triton-launch-mode=remote `

**config.yaml used for profiling**

`model_repository: /tmp/models
run_config_profile_models_concurrently_enable: true
override_output_model_repository: true
run_config_search_mode: quick
run_config_search_max_instance_count: 4
run_config_search_max_concurrency: 2
run_config_search_min_model_batch_size: 1
run_config_search_max_model_batch_size: 4
num_configs_per_model: 2
output_model_repository_path: ./_output
export_path: ./profile_results
profile_models:
  - python_back_model
  - bert_based_model
perf_analyzer_flags:
  input-data: "zero"`

**Additional context**
can you provide how can i pass input data that is required by each individual models inside model repository . what exact is missing in input data .json 

`- MlLogger.inference. - 1240 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.055 - MlLogger.inference. - 1204 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.055 - MlLogger.inference. - 1294 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.055 - MlLogger.inference. - 1205 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1219 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1229 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1246 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1277 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1259 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.056 - MlLogger.inference. - 1301 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1210 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1240 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1204 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1205 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1294 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
2024-05-16 07:05:13.091 - MlLogger.inference. - 1268 - ERROR - [ TRITON SERVICE] context_id=static Error: Exception while parsing input data: {input_error}
I0516 07:05:33.983377 1 model_lifecycle.cc:620] successfully unloaded 'model' version 1`
Thanks.

**Comment by Tabrizian** [comment]
@nv-braf / @matthewkotila  By any chance have you seen this error before?
